### PR TITLE
feat: Swift Cyclomatic Complexity

### PR DESCRIPTION
**Pull request by JessyB**
# Swift Cyclomatic complexity

- [x] Implement a CyclomaticComplexityVisitor
- [x] Call CyclomaticComplexityVisitor from SwiftSensor

# References
- https://github.com/insideapp-oss/sonar-flutter/blob/develop/dart-lang/src/main/java/fr/insideapp/sonarqube/dart/lang/antlr/CyclomaticComplexityVisitor.java
- https://github.com/insideapp-oss/sonar-flutter/blob/develop/dart-lang/src/main/java/fr/insideapp/sonarqube/dart/lang/DartSensor.java

# DoD
- [x] Unit tests
- [x] Documentation (and developer documentation) update

### CODE REVIEW

File: swift-lang/src/main/java/fr/insideapp/sonarqube/swift/lang/antlr/SwiftCyclomaticComplexityVisitor.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    @Override
+    public void fillContext(SensorContext context, AntlrContext antlrContext) {
+        final InputFile file = antlrContext.getFile();
+
+        synchronized (SwiftCyclomaticComplexityVisitor.class) {
+            try {
+                context.<Integer>newMeasure().on(file).forMetric(CoreMetrics.COMPLEXITY).withValue(complexity).save();
+                complexity = 0;
```
Comment (gaelfoppolo) [inline swift-lang/src/main/java/fr/insideapp/sonarqube/swift/lang/antlr/SwiftCyclomaticComplexityVisitor.java:64]: If the saving of the measure fail (throw), the complexity will never be resetted, right?
So maybe this should be outside of the try/catch, or in a finally maybe?

File: swift-lang/src/test/java/fr/insideapp/sonarqube/swift/lang/antlr/SwiftCyclomaticComplexityVisitorTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
+package fr.insideapp.sonarqube.swift.lang.antlr;
```
Comment (gaelfoppolo) [inline swift-lang/src/test/java/fr/insideapp/sonarqube/swift/lang/antlr/SwiftCyclomaticComplexityVisitorTest.java:1]: No header.

**Review by gaelfoppolo — CHANGES_REQUESTED** [review]

File: swift-lang/src/main/java/fr/insideapp/sonarqube/swift/lang/antlr/SwiftCyclomaticComplexityVisitor.java (hunk 1 above)
Comment (JessyB) [reply swift-lang/src/main/java/fr/insideapp/sonarqube/swift/lang/antlr/SwiftCyclomaticComplexityVisitor.java]: You're right. fixed

File: swift-lang/src/test/java/fr/insideapp/sonarqube/swift/lang/antlr/SwiftCyclomaticComplexityVisitorTest.java (hunk 2 above)
Comment (JessyB) [reply swift-lang/src/test/java/fr/insideapp/sonarqube/swift/lang/antlr/SwiftCyclomaticComplexityVisitorTest.java:18]: Fixed

**Review by gaelfoppolo — APPROVED** [review]

**Pull request merged**